.github/workflows/test-all-warehouses.yml

```diff
@@ -51,9 +51,6 @@ jobs:
             dremio,
           ]
         include:
-          # If we're not running on a specific dbt version, then always add postgres on 1.8.0
-          - dbt-version: "${{ inputs.dbt-version || '1.8.0' }}"
-            warehouse-type: postgres
```

Comment:
The tests are no longer compatible with `dbt<1.10.8` (though the package itself still is)